TMS.Net07.Homework.HelloWorld/TMS.Net07.Homework.HelloWorld/Hard.cs

```diff
@@ -1,16 +1,20 @@
 ﻿using System;
 
-class Hard 
+class Hard
 {
-    static void Main () 
+    static void Main()
     {
 
         int number;
-        string enter;
+        string enter, txtMessage = "You are enter";
```

Comment:
```suggestion
        const string txtMessage = "You are enter";
```
These values don't seem to be changed, so suggest making it constant

Reply:
Done.